less/forum/NotificationsDropdown.less

```diff
@@ -35,14 +35,13 @@
   color: @header-bg;
   font-size: 11px;
   font-weight: bold;
-  padding: 2px 4px 3px;
+  padding: 3px 4px;
   line-height: 1em;
   border-radius: 10px;
-  border: 1px solid @header-bg;
+  box-shadow: 0px 0px 0px 1px @header-bg;
```

Comment:
This should probably use the `.box-shadow` mixin just like the other commit did.

Reply:
I made the change, can you check whether it is OK?

Reply:
@TurboProgramming have you tried your edit on a local installation by any chance? I could be wrong, but from my experience mixins are most commonly included as (a kind of) function calls, see http://lesscss.org/#mixins

Reply:
@luceos Unfortunately, no. And I don't think I can at the moment. I'll try my best, but if possible can anyone of you do it?